lib/formatters.js

```diff
@@ -92,9 +92,12 @@ const flatten = (data, prefix = 'data') => {
         });
         return returnVal;
     }
-    return {
-        [`${prefix}`]: formatVal(data)
-    };
+    else if (data || (data === false)) {
```

Comment:
As a nit, `else` is unnecessary after a `return`. Also, this feels wrong; what if `data` is `0` or `NaN`? Seems like you really want

    if (data == null) {
        return {};
    }
    return {
        [`${prefix}`]: formatVal(data)
    }

Reply:
Good catch. I changed this to specifically catch null and undefined.